glotaran/parameter/parameter.py

```diff
@@ -485,22 +485,23 @@ def markdown(
         """
         md = f"{self.full_label}"
 
-        value = f"{self.value:.2e}"
-        if self.vary:
-            if self.standard_error is not np.nan:
-                value += f"±{self.standard_error}"
-            if initial_parameter is not None:
-                initial_value = initial_parameter.get(self.full_label).value
+        parameter = self if all_parameters is None else all_parameters.get(self.full_label)
+        value = f"{parameter.value:.2e}"
+        if parameter.vary:
+            if parameter.standard_error is not np.nan:
+                value += f"±{parameter.standard_error}"
```

Comment:
```suggestion
                value += f"±{parameter.standard_error:.2e}"
```
Currently the standard error is for example rendered like this:
`('s2', 's1'): kinetic.1(5.00e-01±7.284630572763264e-05, initial: 5.00e-01)`

The other option would be to make the significant numbers an option with a sensible default.